libc::write argument type error

When I try to build this as a dependency (cargo new asdf, then adding pyo3-polars = "0.16" to dependencies, then running cargo build) I get a compile error in a libc::write call due to an argument type:

```
...
Compiling polars-parquet v0.42.0
Compiling pyo3-polars v0.16.1
error[E0308]: mismatched types
   --> C:\Users\eizinger\.cargo\registry\src\index.crates.io-6f17d22bba15001f\pyo3-polars-0.16.1\src\alloc.rs:83:78
    |
83  |                 unsafe { libc::write(2, msg.as_ptr() as *const libc::c_void, msg.len()) };
    |                          -----------                                         ^^^^^^^^^ expected `u32`, found `usize`
    |                          |
    |                          arguments to this function are incorrect
    |
```

 I don't really understand it, because when I look up the docs on libc::write, it clearly says that it expects size_t. So maybe something is wrong on my system?

Running on Windows. Toolchain and libc versions below:

```
D:\SW_playground\py03_polars_test>rustup show
Default host: x86_64-pc-windows-msvc
rustup home:  C:\Users\eizinger\.rustup

installed targets for active toolchain
--------------------------------------

i686-pc-windows-msvc
wasm32-unknown-unknown
x86_64-pc-windows-msvc

active toolchain
----------------

stable-x86_64-pc-windows-msvc (default)
rustc 1.80.1 (3f5fd8dd4 2024-08-06)
```

```
D:\SW_playground\py03_polars_test>cargo tree | grep libc
    ├── libc v0.2.156
    │   │   │       ├── libc v0.2.156
    │   ├── libc v0.2.156
    │   │   └── libc v0.2.156
```

I'm not exactly an expert on everything related to compilation, so it's probably something silly because otherwise someone else would've encountered this.
Any help would be appreciated; sorry if this is not the right venue, I don't really know where else to ask.

--- a/pyo3-polars/src/alloc.rs
+++ b/pyo3-polars/src/alloc.rs
@@ -80,7 +80,9 @@ impl PolarsAllocator {
             if r.is_none() {
                 // Do not use eprintln; it may alloc.
                 let msg = b"failed to get allocator capsule\n";
-                unsafe { libc::write(2, msg.as_ptr() as *const libc::c_void, msg.len()) };
+                // Message length type is platform-dependent.
+                let msg_len = msg.len().try_into().unwrap();
+                unsafe { libc::write(2, msg.as_ptr() as *const libc::c_void, msg_len) };
             }
             r.unwrap_or(&FALLBACK_ALLOCATOR_CAPSULE)
         })

